Assignments/task-1/src/question/insert.rs

```diff
@@ -0,0 +1,17 @@
+pub extern crate mysql;
+
+/// Function 'insert_into_db' inserts values to the table in the database
+///
+/// #Arguments
+///
+/// query_test: Is of type String taking the insert query
+/// link: Is of type String
+///
+/// #Return
+///
+/// Returns Option<()>
+pub fn insert_into_db(query_test: String, link: String) -> Option<()> {
```

Comment:
Please incorporate proper error handling and return type should be self-explanatory.